sentry-android-core/build.gradle.kts

```diff
@@ -64,6 +66,27 @@ android {
     }
 }
 
+tasks {
+    val androidJavadocs by creating(Javadoc::class) {
+        source = android.sourceSets["main"].java.getSourceFiles()
```

Comment:
this one looks very different than https://github.com/vanniktech/gradle-maven-publish-plugin/blob/master/src/main/kotlin/com/vanniktech/maven/publish/tasks/AndroidJavadocs.kt#L11-L33

Reply:
Can't we just use this plugin?

Reply:
the plugin does the release too and this should be done locally because of Craft, also signing, so I guess this would be a blocker for the plugin.
if the plugin is able to generate everything locally but not publishing them at all, that would be fine.

Reply:
maybe we can use the plugin for moving away from Bintray/JCenter and later spend more time splitting up the things (signing and publishing locally), @bruno-garcia ?